openmp/libomptarget/include/Shared/Debug.h

```diff
@@ -187,4 +189,13 @@ inline uint32_t getDebugLevel() {
     }                                                                          \
   } while (false)
 
+#define DUMP_INFO(always, _flags, _id, ...)                                    \
+  do {                                                                         \
+    if (always) {                                                              \
```

Comment:
Why do we need this? The old interface was always guarded by `if (get_info() == ...)` so why do we need this `always` field now?

Reply:
I'm not  quite sure what `get_info` you're referring to. From what I  can see there's only `getDebugLevel()` and `getInfoLevel()` which are used to check if the specific environment variables are set. The extension ompx_dump_mapping_tables is supposed to work  without the `LIBOMPTARGET_INTO` or  `LIBOMPTARGET_DEBUG` variables set.